app/src/models/repository.ts

```diff
@@ -213,3 +213,15 @@ export function getForkContributionTarget(
     ? repository.workflowPreferences.forkContributionTarget
     : ForkContributionTarget.Parent
 }
+
+/**
+ * Returns whether the fork is contributing to the parent
+ */
+export function isRepositoryAForkContributingToParent(
```

Comment:
Just a suggestion (I try to avoid having `A` in identifiers 😳 )
```suggestion
export function isForkedRepositoryContributingToParent(
```